frontend/packages/dev-console/src/components/import/advanced/AdvancedSection.tsx

```diff
@@ -83,12 +94,19 @@ const List: React.FC<AdvancedSectionProps> = ({ appResources, values }) => {
 
 const AdvancedSection: React.FC<AdvancedSectionProps> = ({ values, appResources }) => {
   const { t } = useTranslation();
+  const [resourceType] = useResourceType();
+  const { setFieldValue } = useFormikContext<FormikValues>();
+  const formPage = window.location.pathname.split('/')[1];
+
+  React.useEffect(() => {
+    !['edit', 'knatify'].includes(formPage) && setFieldValue('resources', resourceType);
```

Comment:
we can do it but is there a possibility to get this information from formik values and check based on that?

Reply:
Done.